doc/manual/src/language/identifier.md

```diff
@@ -0,0 +1,40 @@
+# Identifier
+
+An *identifier* is an [ASCII](https://en.wikipedia.org/wiki/ASCII) character sequence that:
+- Starts with a letter (`a-z`, `A-Z`) or underscore (`_`)
+- Can contain any number of:
+  - Letters (`a-z`, `A-Z`)
+  - Digits (`0-9`)
+  - Underscores (`_`)
+  - Apostrophes (`'`)
+  - Hyphens (`-`)
+- Is not one of the [keywords](#keywords)
+
+> **Syntax**
```

Comment:
I was using the `**Grammar**` header for sections like this that are written in BNF pseudocode, reserving `**Syntax**` for sections written in Nix pseudocode.

Reply:
The other sections not being even close to BNF is merely an implementation error.